app/pages/view-application.tsx

```diff
@@ -23,6 +24,8 @@ class ViewApplication extends Component<Props> {
         }
         ...ApplicationDetailsContainer_query
           @arguments(applicationId: $applicationId)
+        ...RequestedChangesByFormResult_query
+          @arguments(applicationId: $applicationId)
```

Comment:
@wenzowski Instead if passing that `applicationId` argument to the fragments, should we have 

```
application(id: $applicationId) {
  ...ApplicationDetailsContainer_application
  ...RequestedChangesByFormResult_application
}
```

And change `ApplicationDetailsContainer` and `RequestedChangesByFormResult` to be fragments on `Application`

Reply:
^ absolutely

Reply:
this will require a refactor a bunch of views and containers. I'd rather have this under a separate refactor task